kratos/geometries/hexahedra_3d_27.h

```diff
@@ -17,11 +17,13 @@
 #pragma once
 
 // System includes
+#include <numeric>
 
 // External includes
 
 // Project includes
 #include "geometries/quadrilateral_3d_9.h"
+#include "geometries/triangle_3d_3.h"
```

Comment:
We should redefine some methods in an external file, aka utilities, in a static manner and depending on the coordinates to avoid constructing geometries (which is quite expensive)